### PR TITLE
Capture container image size

### PR DESCRIPTION
**Pull request by bissenbay**

### CODE REVIEW

File: thoth/package_extract/image.py [hunk 1]
```diff
@@ -515,6 +515,15 @@ def download_image(
     _LOGGER.debug("%s stdout: %s", _SKOPEO_EXEC_PATH, stdout)
 
 
+def get_image_size(path: str) -> int:
```
Comment (fridex) [inline thoth/package_extract/image.py]: Please add a docstring so coala is happy.

**Comment by goern** [comment]
Do we require a Thoth.storages change do that the data is synced into the knowledge graph?

**Comment by fridex** [comment]
> Do we require a Thoth.storages change do that the data is synced into the knowledge graph?

Yes, the syncing method for package-extract results needs to be adjusted and a new schema migration generated.

File: thoth/package_extract/image.py (hunk 1 above)
Comment (bissenbay) [reply thoth/package_extract/image.py]: > Please add a docstring so coala is happy.

Done!

Comment (bissenbay) [reply thoth/package_extract/image.py]: > Also, please check if the computed image size matches what `docker inspect` reports.

@fridex I pulled the same image using `docker pull` and the size I see is 999 MB. But, if I go to https://quay.io/repository/thoth-station/thoth-coala?tab=tags and see the size is 321.9 MB which is closer to what we getting in bytes `337560925`. WDYT?

Comment (fridex) [reply thoth/package_extract/image.py]: Sounds legit, also see https://github.com/moby/moby/issues/31298

**Comment by fridex** [comment]
> Build failed.
> 
> * [thoth-coala ](https://managesf.thoth-station.ninja/logs/03/303/f281f15f204d82638580f35ddc4f4a3539b228ac/check/thoth-coala/1d2d931/) : FAILURE in 58s
> * [thoth-pytest ](https://managesf.thoth-station.ninja/logs/03/303/f281f15f204d82638580f35ddc4f4a3539b228ac/check/thoth-pytest/c2bd571/) : SUCCESS in 3m 59s

Coala is still not happy:

```
thoth/package_extract/image.py
| 518| def·get_image_size(path:·str)·->·int:
|    | [NORMAL] PyDocStyleBear:
|    | D400: First line should end with a period (not 'e')
```

https://media.giphy.com/media/7p3e2WCM0VEnm/giphy.gif

**Comment by bissenbay** [comment]
> > Build failed.
> > 
> > * [thoth-coala ](https://managesf.thoth-station.ninja/logs/03/303/f281f15f204d82638580f35ddc4f4a3539b228ac/check/thoth-coala/1d2d931/) : FAILURE in 58s
> > * [thoth-pytest ](https://managesf.thoth-station.ninja/logs/03/303/f281f15f204d82638580f35ddc4f4a3539b228ac/check/thoth-pytest/c2bd571/) : SUCCESS in 3m 59s
> 
> Coala is still not happy:
> 
> ```
> thoth/package_extract/image.py
> | 518| def·get_image_size(path:·str)·->·int:
> |    | [NORMAL] PyDocStyleBear:
> |    | D400: First line should end with a period (not 'e')
> ```
> 
> sad coala



> > Build failed.
> > 
> > * [thoth-coala ](https://managesf.thoth-station.ninja/logs/03/303/f281f15f204d82638580f35ddc4f4a3539b228ac/check/thoth-coala/1d2d931/) : FAILURE in 58s
> > * [thoth-pytest ](https://managesf.thoth-station.ninja/logs/03/303/f281f15f204d82638580f35ddc4f4a3539b228ac/check/thoth-pytest/c2bd571/) : SUCCESS in 3m 59s
> 
> Coala is still not happy:
> 
> ```
> thoth/package_extract/image.py
> | 518| def·get_image_size(path:·str)·->·int:
> |    | [NORMAL] PyDocStyleBear:
> |    | D400: First line should end with a period (not 'e')
> ```
> 
> sad coala

@fridex :laughing: this memo made my day

**Pull request merged**